### ISSUE TITLE
Bug: Neither hi_res nor fast partition strategies classify section title as Title

### ISSUE THREAD

**Issue by kubni** [Unstructured-IO/unstructured#3783]
Hello. In my PDF, I have 3 pages that are currently of interest to me. For reference, those are pages 63, 64, 65.
On the page 63, there is a title "7. Institutional stakeholders and their roles", after which those stakeholders and their descriptions are listed until the end of page 65.
However, whichever combination of chunking hyperparameters I tried didn't make all of them be in the same chunk. 

I tried different values for `chunk_max_characters` and `chunk_new_after_n_chars`, as well as enabled `chunk_multipage_sections`, paired with `by_title` chunking strategy, but to no avail.

Is there a common way of solving this problem?

EDIT: Forgot to say that I am using the free trial of Unstructured Serverless API via the provided Python unstructured-ingest library.

**Comment by kubni** [comment]
I managed to get them all in the same chunk by setting `chunk_max_characters` and `chunk_new_after_n_chars` to `20000` (although I tried keeping max at 2000 and changing new to lower values and it didn't change anything), however the chunk doesn't start with that title, even though `by_title` chunking strategy was used, and it even includes a title from page 66.

**Comment by kubni** [comment]
I noticed why `by_title` includes the next section's title in the previous section's chunk.
For some reason, neither `hi_res` (tried both `layout_v1.1.0` and `yolox`) nor `fast` partition strategies classify the title as a Title.
Hi_res models classify the whole page as an `Image`, while fast strategy classifies the title as a `ListItem` (probably due to its format being like `8. Title`).

Here is a screenshot of the problematic page:
![img](https://github.com/user-attachments/assets/96d2bddf-9ebc-4d49-a59d-d2bb479b03a1)

**Comment by micmarty-deepsense** [comment]
This output seemed off to me, so I checked with someone from another team, and as far as I understand, it's a known but unresolved issue. 

It appears the Object Detection model likely picks up the title correctly, but then it gets lost in the **post-processing** steps. Because the analysis mode only shows the output after these steps, we're not seeing it in the "od_model" images you sent over in that zip file.

Also, the title starting with "8. " might be getting misidentified as a ListItem.
In summary - I'm going to bring this to the attention of the ML team to explore future solutions. 

Thanks for reporting that!

**Comment by kubni** [comment]
Is it possible to turn off the post processing? Or alter it somehow?

**Comment by scanny** [comment]
@kubni in general, chunking cannot guarantee what will appear in the _same_ chunk, it can only guarantee what will appear in _different_ chunks. This is the notion of _segregation guarantees_.

What gets placed in the same chunk is determined solely by how much room is left in the current chunk and how big the next element is.

The segregation guarantee for `by_title` is that two paragraphs from different document sections (under two different titles) will not appear in the same chunk.

Note that if you use the `combine_text_under_n_chars` parameter then segregation guarantees are ignored, because "combining" is combining chunks, not elements, so happens after chunking.

To get a feel for it, you can set `combine_text_under_n_chars` to `0` and then change your `max_characters` setting. With `by_title` what you should observe is that each `Title` element starts a new chunk, and if your `max_characters` setting is large enough to fit them, then paragraphs following a title will appear in the same chunk as the title.

**Comment by kubni** [comment]
@scanny Yeah, I managed to get to the point where chunks start with the title. The bigger problem is that a title from that image up there isn't recognized as a title, but Michal told me that its a problem with the post-processing steps.

**Comment by scanny** [comment]
@kubni I'm closing this as I think it is resolved, if not actually "fixed". Please feel free to reopen or open a new ticket if there is something actionable we haven't addressed :)
